tests/RequestDataIdentificationTest.php

```diff
@@ -26,24 +26,32 @@
 test('header identification works', function () {
     InitializeTenancyByRequestData::$header = 'X-Tenant';
     $tenant = Tenant::create();
-    $tenant2 = Tenant::create();
 
     $this
         ->withoutExceptionHandling()
-        ->get('test', [
-            'X-Tenant' => $tenant->id,
-        ])
+        ->withHeader('X-Tenant', $tenant->id)
+        ->get('test')
         ->assertSee($tenant->id);
 });
 
 test('query parameter identification works', function () {
     InitializeTenancyByRequestData::$queryParameter = 'tenant';
 
     $tenant = Tenant::create();
-    $tenant2 = Tenant::create();
 
     $this
         ->withoutExceptionHandling()
         ->get('test?tenant=' . $tenant->id)
         ->assertSee($tenant->id);
 });
+
+test('cookie identification works', function () {
+    InitializeTenancyByRequestData::$header = 'X-Tenant';
```

Comment:
Why do we set the `$header` in a cookie-related test?

Reply:
It'd make more sense to add a new parameter, just like the other two methods have:
https://github.com/archtechx/tenancy/blob/693e00b9be9daf7fb3ada777d5a510f7ec424950/src/Middleware/InitializeTenancyByRequestData.php#L14-L15

Reply:
> It'd make more sense to add a new parameter, just like the other two methods have:

I was thinking of adding the separate property but then didn't and thought it would be unnecessary. But I think the separate property will make more sense.